bin/compile

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+[ "$BUILDPACK_XTRACE" ] && set -o xtrace
+
+set -e
+
+export_env_dir() {
+  env_dir=$3
+  blacklist_regex=${3:-'^(PATH|GIT_DIR|CPATH|CPPATH|LD_PRELOAD|LIBRARY_PATH)$'}
+  if [ -d "$env_dir" ]; then
+    for e in $(ls $env_dir); do
+      echo "$e" | grep -qvE "$blacklist_regex" &&
+      export "$e=$(cat $env_dir/$e)"
+      :
```

Comment:
I have shamelessly copied and pasted this from Jamie's known-working script here https://github.com/dabapps/heroku-buildpack-django-topography/blob/master/bin/compile#L3-L14

The colon is actually a builtin command which returns a success exit code. I'm not certain why that would be required here.